spec/venue_spec.rb

```diff
@@ -44,12 +44,45 @@
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
+
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+
+  describe '#over_capacity' do
+    it 'returns false if less patrons then capacity' do
+
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      expect(venue.over_capacity?).to be false
+    end
+
+    it 'returns true if more patrons then capacity' do
+      venue = Venue.new('Bluebird', 4)
+      venue.add_patron('Mike')
+      venue.add_patron('Megan')
+      venue.add_patron('Bob')
+      venue.add_patron('James')
+      venue.add_patron('Cat')
+      expect(venue.over_capacity?).to be true
+    end
+  end
+  describe '#kick_out' do
+    it 'removes patrons from venue if over capacity' do
```

Comment:
Really make sure that you are following the interaction patterns as it is written. Where did you include the `venue.kick_out` assertion?

And the expect was
```
pry(main)> venue.over_capacity?
=> false
```
so your last method failed because you expected 4 instead of false